Library/Formula/icecream.rb

```diff
@@ -0,0 +1,92 @@
+class Icecream < Formula
+  homepage ""
+  url "https://github.com/icecc/icecream/archive/1.1rc1.tar.gz"
```

Comment:
it's a release candidate, but it compiles on osx without modification unlike 1.01.